### PR TITLE
Fix crash

### LINKED ISSUE #22: Crash when opening up browser then story
Example URL to assist with testing: https://lobste.rs/s/3qgyzp

Be in that story, then open the lobsters link in the comment in the web view. Then use the share sheet to in the webview to open the lobster link again but in the app using "Open in claw". None of the links then work and when you exit the view controller it crashes

When in a story, there is also a crash if you try to use the share sheet on a story and use the "Open in claw" action to open the same story. These are most likely related.

### PR DESCRIPTION
**Pull request by WhollySoftware**
Crash when opening up browser then story #22 -> This is fixed.

Closes #22

### CODE REVIEW

File: GitPush.command [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+cd $(dirname "$BASH_SOURCE")
+git add .
+git status
+git commit -m 'Crash when opening up browser then story #22 -> This is fixed.'
```
Comment (twodayslate) [inline GitPush.command:6]: Remove this file. Also see https://chris.beams.io/posts/git-commit/

File: claw/ContentView.swift [hunk 2]
```diff
@@ -131,6 +131,7 @@ struct ContentView: View {
             }).environmentObject(settings).environment(\.managedObjectContext, viewContext)
         }.environment(\.didReselect, didReselect.eraseToAnyPublisher()).accentColor(settings.accentColor).font(Font(.body, sizeModifier: CGFloat(settings.textSizeModifier))).onOpenURL(perform: { url in
             let _ = print(url)
+            UIApplication.shared.windows.first?.rootViewController?.presentedViewController?.dismiss(animated: true, completion: nil)
```
Comment (twodayslate) [inline claw/ContentView.swift:134]: Can you add a check that the there is a sheet to dismiss? If self.observableSheet.sheet is nil? also only dismiss if you are about to present a new sheet. Can you just set self.observableSheet.sheet to nil to dismiss? If not set self.observableSheet.sheet in the completion block?

**Review by twodayslate — CHANGES_REQUESTED** [review]
What was changed in .../project.xcworkspace/xcuserdata/gauravgudaliya.xcuserdatad/UserInterfaceState.xcuserstate?

File: claw/ContentView.swift (hunk 2 above)
Comment (twodayslate) [reply claw/ContentView.swift:134]: Sadly I believe it is impossible in SwiftUI to have unlimited sheets (just keep adding one on top)

Comment (WhollySoftware) [reply claw/ContentView.swift:134]: there is it's possible unlimited sheets but problem is when you click share then we are not able to identify the presented sheet that's is the issue so when we presenting new sheet first I will dismiss current presented all sheet then open new sheet

Comment (twodayslate) [reply claw/ContentView.swift:134]: Can't we just keep track of the top then for the share sheet?

Comment (twodayslate) [reply claw/ContentView.swift:134]: Let's create a ticket for unlimited sheets and just fix this bug for now

Comment (WhollySoftware) [reply claw/ContentView.swift:134]: ok sir

Comment (twodayslate) [reply claw/ContentView.swift:134]: The original comment still applies.

**Comment by twodayslate** [comment]
Remove the .DS_Store files and then this looks good to me

**Review by twodayslate — APPROVED** [review]

**Pull request merged**